pkg/operator/sync_signer_v311_00.go

```diff
@@ -108,6 +112,13 @@ func manageSigningSecret_v311_00_to_latest(client coreclientv1.SecretsGetter) (*
 	secret := resourceread.ReadSecretV1OrDie(v310_00_assets.MustAsset("v3.10.0/service-serving-cert-signer-controller/signing-secret.yaml"))
 	existing, err := client.Secrets(secret.Namespace).Get(secret.Name, metav1.GetOptions{})
 	if !apierrors.IsNotFound(err) {
+		certs, err := cert.ParseCertsPEM(existing.Data["tls.crt"])
```

Comment:
Not sure what's all the errors that may occur during `Get()`, but I suppose if there was an error retrieving the secret which was different from `NotFound`, `existing` may be `nil`.